tests/kuttl/tests/fernet_rotation/02-assert.yaml

```diff
@@ -0,0 +1,6 @@
+# validate that token saved in previous step works
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+namespaced: true
+commands:
+  - script: $KEYSTONE_KUTTL_DIR/../common/scripts/validate_test_token.sh
```

Comment:
starting with kuttl 0.17.0 should be relative to the test dir, so `../../common/scripts/validate_test_token.sh`. otherwise you could not run it manually from the base dir